### ISSUE TITLE
Schedule new Updates not possible (category and products)

### ISSUE THREAD

**Issue by dimaMackert** [Smile-SA/elasticsuite#1741]
### Preconditions

Magento Version : 2.3.4 EE (also on 2.3.3)

ElasticSuite Version : 2.8.4

Environment : Dev

Third party modules : N/A

### Steps to reproduce
1. Login into the Backend
2. Navigate to Catalog -> Categories. Be sure, you activated the Default Category
3. Click on the Schedule New Update button.

### Expected behavior
- You have the possibilities to edit the update options.

### Behavior without patch
- The error-message "Attention Something went wrong." appears.

### More info
The message appears in dev mode. In prod. mode you will be redirected to the homepage.
I fixed (workaround) the issue by doing the following: 

\Smile\ElasticsuiteCatalog\Plugin\Ui\Category\Form\DataProviderPlugin.php

```
public function aroundGetMeta(CategoryDataProvider $dataProvider, \Closure $proceed)
    {
        $meta = $proceed();

        $currentCategory = $dataProvider->getCurrentCategory();

        if ($currentCategory->getId() === null || $currentCategory->getLevel() < 2) {
            $meta['display_settings']['children']['facet_config']['arguments']['data']['config']['visible'] = false;
            $meta['display_settings']['children']['facet_config']['arguments']['data']['config']['componentType'] = 'field';
            $meta['display_settings']['children']['facet_config']['arguments']['data']['config']['formElement'] = 'input';
        }

        return $meta;
    } 

```

**Comment by rbayet** [comment]
Hello @dimaMackert,

Thanks for the issue and the solution.
I've tested the fix and indeed those two news lines are indeed required and enough to fix the problem.

Would you like to push us a PR with a commit comment following the format "Fixes #1741 [your comment]" ?

Regards,

**Comment by rbayet** [comment]
Thanks @dimaMackert !
